example/webpack.config.js

```diff
@@ -69,6 +69,11 @@ module.exports = {
     publicPath: '/',
     filename: 'rnw.bundle.js',
   },
+  devServer: {
+    headers: {
+      'Document-Policy': 'js-profiling',
+    },
+  },
```

Comment:
it seems that people need to add this to their webpack config to test in debug mode? I think we should add this to the documentation?
Also, have you tested in release mode? (as this only seems to be set for the `devServer`)

Reply:
I added a note in README.

> Also, have you tested in release mode? (as this only seems to be set for the devServer)

@hannojg No, not yet. I think the best way here would be to try to integrate that in Expensify codebase and if we get any useful insights, then update README?

Reply:
okay